Essentials/src/main/java/com/earth2me/essentials/EssentialsUpgrade.java

```diff
@@ -1,8 +1,9 @@
 package com.earth2me.essentials;
```

Comment:
In the long-term, we could maybe consider using Configurate's transform stuff? Although some of these migrations are very cursed, and not sure it fits exactly with how we currently do data upgrades.